package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "aws-lambda-ric",
-  "version": "3.2.0",
```

Comment:
This looks like a leftover from a previous bump? Maybe we should clearly state in the readme that package-lock should be commited

Reply:
Yes, Sure let me add it to the README.

Reply:
Updated README and the PR template.